src/core/model_config.proto

```diff
@@ -530,6 +530,25 @@ message ModelOptimizationPolicy
     repeated Accelerator cpu_execution_accelerator = 2;
   }
 
+  //@@
+  //@@  .. cpp:var:: message IndirectPinnedBufferSetting
+  //@@
+  //@@     Specify whether to use a pinned memory buffer when transferring data
+  //@@     between non-pinned system memory and device memory. i.e. when request
```

Comment:
"...and GPU memory. Using a pinned memory buffer for system to/from GPU transfers will typically provide increased performance. When the request..."